cmake/sanitize.cmake

```diff
@@ -9,6 +9,16 @@ option (SANITIZE "Enable one of the code sanitizers" "")
 set (SAN_FLAGS "${SAN_FLAGS} -g -fno-omit-frame-pointer -DSANITIZER")
 
 if (SANITIZE)
```

Comment:
afaiu, coverage doesn't fall into this category. IMO would make sense to check include it too